src/imitation/rewards/reward_nets.py

```diff
@@ -163,78 +208,35 @@ def dtype(self) -> th.dtype:
             return th.get_default_dtype()
 
 
-class ShapedRewardNet(RewardNet):
-    """A RewardNet consisting of a base net and a potential shaping."""
+class RewardNetWrapper(RewardNet):
+    """An abstract RewardNet wrapping a base network.
+
+    A concrete implementation of `forward` method is needed.
+    Note: by default, `predict`, `predict_th`, `preprocess`, `predict_processed`,
+    `device` and all the PyTorch `nn.Module` methods will be inherited from `RewardNet`
+    and not passed through to the base network. If any of these methods is overridden
+    in the base `RewardNet`, it will be not inherited from `RewardNetWrapper`.
```

Comment:
Inherited has a technical meaning here that isn't appropriate (as it's a composition relationship)
```suggestion
    in the base `RewardNet`, this will not affect `RewardNetWrapper`.
```